### ISSUE TITLE
[Epic] CDK: Layers of Abstraction and Publishing

### ISSUE THREAD

**Issue by theyueli** [airbytehq/airbyte#43958]

**Comment by postamar** [comment]
I think we can declare victory here. Going over the DoD:

> * Split new cdk into base/bulk-load/bulk-extract and publish all seperatly

This has been done. Furthermore, I just finished rewriting the JDBC toolkit and it's now in a state that I'm happy about.

> * Move job runner to lowest level

Won't do. We've agreed with destinations to factor common scheduling patterns only after they've come up with a destinations-specific scheduler of their own.

> * coordinate with destinations team on implementation of new partition executers

We've coordinated with the destinations team. 

> * remove ourselves from destination's critical path towards their new CDK development.

Done.

**Comment by theyueli** [comment]
wonderful!
